kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -203,19 +210,8 @@ var TextView = BaseView.extend({
         return this;
     },
 
-    text_changed: function() {
-        this.model.set(this.text_attr, this.$(".rating-text-feedback")[0].value);
-    },
-
-    toggle_disabled: function(val) {
-        if( typeof val === "undefined" ) {
-            val = !this.is_disabled();
-        }
-        this.$(".rating-text-feedback").attr("disabled", val);
-        return this;
-    },
-
-    is_disabled: function() {
-        return this.$(".rating-text-feedback").attr("disabled") === "disabled";
-    }
+    text_changed: _.throttle(function() {

```

Comment:
This was the particular one where debounce might be better. Then, it'll wait until they stop typing for a bit to save, rather than pounding the server constantly. I'd recommend debounce with 2000ms.